src/app/pages/root/map/map-component.jsx

```diff
@@ -105,7 +105,11 @@ class MapComponent extends PureComponent {
 
   setMapTerrain = (terrainCameraOffset, { cellId, coordinates }) => {
     const { query } = this.props;
-    const activeLayers = query.activeLayers ? query.activeLayers : null;
+    const activeLayers = query.activeLayers
+      ? query.activeLayers.filter(
+        l => l.layerCategory !== 'terrestrial' && l.layerCategory !== 'aquatic'
```

Comment:
@benlaken maybe in the future, if this filtering gets growing, would be nice to have a param on layers table as `isVisibleOnTerrain`?